examples/structured_2d_dgsem/elixir_shallowwater_source_terms.jl

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
Okay, I will ping you later to double check `elixir_shallowwater_well_balanced.jl` and `elixir_shallowwater_ec.jl` because it uses a somewhat hacky strategy to set discontinuities.

Reply:
@ranocha It would be good to have a look at the updated `elixir_shallowwater_well_balanced.jl` and `elixir_shallowwater_ec.jl` files

Reply:
Please request my review again